docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
 location
 ========
 
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| Predicate                         | Value Type     | Range (if needed) | Usage Notes                                                             |
++===================================+================+===================+=========================================================================+
+| relators:rps                      | Literal or URI |                   | Use for <mods:physicalLocation> values, preferably using                |
+|                                   |                |                   | a URI for the organization from a controlled vocabulary                 |
+|                                   |                |                   | such as VIAF of Library of Congress Real World Objects.                 |
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| skos:note                         | Literal        |                   | Use to note <mods:shelfLocator> strings.                                |
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| dbo:collection                    | Literal        |                   | Use to note <mods:physicalLocation[@displayLabel=Collection]> strings.  |
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+
+physicalLocation as URI
+-----------------------
+
+Use Case
+^^^^^^^^
+Many of records have valueURI attributes set for physicalLocation. This is inconsistent, even in our own collections.'
+
+Justification
+^^^^^^^^^^^^^
+When available, we will opt to use valueURI values as the URI value for relators:rps, to better qualify objects of relators:rps.
+
+Xpath
+^^^^^
+
+`mods:location/mods:physicalLocation[@valueURI=https://example.org/authority/1]`
```

Comment:
Doesn't hurt to include the example.org URI, but isn't consistently used across the group submissions. I think it might be easier to just include the presence of the valueURI attribute.